### ISSUE TITLE
Have a rule to force new lines between "select" and "from"

### ISSUE THREAD

**Issue by thisiscab** [sqlfluff/sqlfluff#4868]
### Search before asking

- [X] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### Description

We want a rule where a new line would be added after the "select" and before the "from" statement. Here is a concrete example:

From (bad)
```sql
select
    column_a,
    column_b,
    column_c
from potato
```

To (good)
```sql
select

    column_a,
    column_b,
    column_c

from potato
```

### Use case

We have a document highlighting all the different rules we want our DBT codebase to look like. Most of the rules that we have in place can be represented and automated via SQLFluff. Unfortunately, this rule isn't something that can be configured through SQLFluff.  

### Dialect

It's not related to a specific dialect, but for we're using Snowflake.

### Are you willing to work on and submit a PR to address the issue?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)

**Comment by peterbud** [comment]
Hi @thisiscab , thanks for opening this issue.

Please check our CONTRIBUTING.md:
> Narrow rules for isolated organizational preferences will not normally be accepted in core. For those cases, please use the plugin architecture.
